test/webgl/texture.spec.js

```diff
@@ -214,3 +214,66 @@ test('WebGL2#Texture2D setParameters', t => {
 
   t.end();
 });
+
+test('WebGL#Texture2D NPOT Workaround: texture creation', t => {
+  const {gl} = fixture;
```

Comment:
Test cases should be run both on WebGL and WebGL2 context (only available in `test-browser`).

There should be some negative test cases, i.e. that test that the constructor throws exceptions when given NPOT textures under WebGL1.